src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -0,0 +1,351 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0" DefaultTargets="Build">
+   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.legacy.props" />
+   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.common.props')" />
+   <Import Project="$(VSSDKRoot)\build\Microsoft.VSSDK.BuildTools.props" />
+   <PropertyGroup>
+      <Shipping>true</Shipping>
+      <ResolveNuGetPackages>true</ResolveNuGetPackages>
+      <PackagesDirectory>$(UserProfile)\.nuget\packages</PackagesDirectory>
+      <TargetFrameworkProfile />
+      <ProjectGuid>{3B96F91B-3B58-40ED-B06E-5CD133A79A63}</ProjectGuid>
+      <ProjectTypeGuids>{82b43b9b-a64c-4715-b499-d71e9ca2bd60};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+      <OutputType>Library</OutputType>
+      <AppDesignerFolder>Properties</AppDesignerFolder>
+      <RootNamespace>NuGet.VisualStudio.Client</RootNamespace>
+      <GeneratePkgDefFile>false</GeneratePkgDefFile>
+      <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
+      <IncludeCopyLocalReferencesInVSIXContainer>true</IncludeCopyLocalReferencesInVSIXContainer>
+      <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
+      <IncludeDebugSymbolsInLocalVSIXDeployment>true</IncludeDebugSymbolsInLocalVSIXDeployment>
+      <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+      <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+      <DeployExtension>false</DeployExtension>
+      <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+      <AssemblyName>NuGet.Tools</AssemblyName> <!-- This name is only temporary, it should be renamed to something better to avoid the confusion-->
```

Comment:
It would explode the complexity of the change. 
I think the renaming would be a task on it's own because the build scripts will need to be changed and we need to analyze further that there are not any other dependencies.